src/codesign-validator.ts

```diff
@@ -46,19 +46,8 @@ export default class CodeSignValidator {
 
     validateMacSignature(fileExtension: string) {
         let promise = utility.defer();
-        let command = `spctl --assess -vv ${this.filePath} --type `;
+        let command = `codesign -dv --verbose=4 ${this.filePath}`;
```

Comment:
i think codesign didn't work for dmg. Let us validate that and add spctl back only for dmg.